plugins/.goreleaser.template.yaml

```diff
@@ -10,21 +10,11 @@ archives:
       - policies/**
 checksum:
   name_template: "checksums.txt"
-  ids:
-    - default
 changelog:
   sort: asc
   filters:
     exclude:
       - "^docs:"
       - "^test:"
-signs:
-  - artifacts: checksum
-    args:
```

Comment:
This should be `args: ["--batch", "-u", "{{ .Env.GPG_FINGERPRINT }}", "--output", "${signature}", "--detach-sign", "${artifact}"]` for the key from GitHub actions secrets to be used